src/postgres_mcp/server.py

```diff
@@ -336,17 +336,18 @@ async def explain_query(
         "Takes longer but provides more accurate information.",
         default=False,
     ),
-    hypothetical_indexes: list[dict[str, Any]] | None = Field(
-        description="""Optional list of hypothetical indexes to simulate. Each index must be a dictionary with these keys:
+    hypothetical_indexes: list[dict[str, Any]] = Field(
+        description="""A list of hypothetical indexes to simulate. Each index must be a dictionary with these keys:
```

Comment:
See comment below, I think I could keep it optional.

Reply:
That was the main fix to not have it as optional.